script.js

```diff
@@ -260,4 +260,36 @@ document.getElementById('lg6-4-m').innerHTML = project4m.projectLanguages.lg6;
 
 document.getElementById('live-4-m').href = project4m.projectLive;
 
-document.getElementById('source-4-m').href = project4m.projectSource;
\ No newline at end of file
+document.getElementById('source-4-m').href = project4m.projectSource;
+
+// Validation form
+
+const form = document.getElementById('form');
+const myemail = document.getElementById('myemail');
+
+function setErrorFor(input, message) {
+  const small = document.getElementById('small1');
+  small.innerText = message;
+  small.classList.add('error');
+}
+
+function isEmail(email) {
+  return /^(([^<>()\]\\.,;:\s@"]+(\.[^<>()\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/.test(email);
+}
+
+function checkEmail() {
+  const myemailValue = myemail.value.trim();
+
+  if (myemailValue === '') {
+    setErrorFor(myemail, 'Email cannot be blank');
+  } else if (myemailValue.toLowerCase() !== myemailValue) {
+    setErrorFor(myemail, 'All character in the email should be lower case');
+  } else if (!isEmail(myemailValue)) {
+    setErrorFor(myemail, 'Email is not valid');
+  }
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  checkEmail();
+});
```

Comment:
- [ ] Nice job :clap: on making this I can see an alert message when the user submits incorrect data. however, your form is not submitted when the user corrects that error. Kindly make sure your form is submitted when no email error validation is found. 
- [ ] Also I kindly recommend removing the alert message when the user corrects the validation error.